platform/solr/solr-factory/src/test/groovy/org/codice/solr/factory/impl/SolrClientAdapterSpec.groovy

```diff
@@ -22,23 +22,19 @@ import org.apache.solr.client.solrj.SolrServerException
 import org.apache.solr.client.solrj.response.SolrPingResponse
 import org.apache.solr.common.SolrException
 import org.apache.solr.common.util.NamedList
+import org.codice.junit.DeFinalize
+import org.codice.junit.DeFinalizer
 import org.codice.solr.client.solrj.UnavailableSolrException
 import org.codice.solr.factory.impl.SolrClientAdapter.Creator
-import org.codice.spock.extension.ClearInterruptions
-import org.codice.spock.extension.DeFinalize
-import org.codice.spock.extension.Supplemental
-import org.codice.spock.extension.builtin.DeFinalizer
+import org.codice.spock.ClearInterruptions
+import org.codice.spock.Supplemental
 import org.junit.runner.RunWith
 import org.spockframework.mock.runtime.MockInvocation
 import spock.lang.Shared
 import spock.lang.Specification
 import spock.lang.Unroll
 
-import java.util.concurrent.Callable
-import java.util.concurrent.CancellationException
-import java.util.concurrent.CompletableFuture
-import java.util.concurrent.Future
-import java.util.concurrent.ScheduledExecutorService
+import java.util.concurrent.*
```

Comment:
❓Any reason we replaced the specific classes with `*`? Pretty sure our standard is to use explicit exports.

Reply:
wow! I did not do that. I wonder how it got in.

Reply:
Evil IDE